.github/CODEOWNERS

```diff
@@ -11,27 +11,27 @@
 # /docs/platforms/apple/ @getsentry/team-mobile-core
 # /docs/platforms/dart/ @getsentry/team-mobile-cross-platform
 # /docs/platforms/react-native/ @getsentry/team-mobile-cross-platform
-# /docs/platforms/unity/ @getsentry/team-mobile-cross-platform
+# /docs/platforms/unity/ @getsentry/gdx
 # /docs/platforms/javascript/guides/cordova/ @getsentry/team-mobile-cross-platform
 
-# /docs/platforms/dotnet/ @getsentry/team-mobile-cross-platform
-# /docs/platforms/java/ @getsentry/team-mobile-core
+# /docs/platforms/dotnet/ @getsentry/team-web-sdk-backend
```

Comment:
I can imagine we'd like `dotnet` also to be `gdx`.
And we play also want to add `godot`, `nintendo-switch`, `playstation`, `powershell`, `unreal` and `xbox`, too.

I'll follow up on that, push to the PR and resolve this comment.

Reply:
@Flash0ver kk - if you need more time we can also merge this and you can open a separate PR, lmk

Reply:
done ... I "grouped" team GDX repo in a block ... feel free to shuffle things around if grouping/order could be made more clear differently